resources/web/wwi/X3dScene.js

```diff
@@ -111,9 +111,15 @@ export default class X3dScene {
         const loader = new Parser(prefix);
         await loader.parse(xmlhttp.responseText, renderer);
         onLoad();
+      } else if (xmlhttp.status === 404) {
+        if (document.getElementById('webotsProgressMessage'))
+          document.getElementById('webotsProgressMessage').innerHTML = 'File not found';
       }
     };
-    xmlhttp.onerror = document.getElementById('webotsProgressMessage').innerHTML = 'File not found.';
+    xmlhttp.onerror = () => {
+      if (document.getElementById('webotsProgressMessage'))
+        document.getElementById('webotsProgressMessage').innerHTML = 'An error occured during the loading.';
```

Comment:
Don't we have an idea about this error? Is there an error string somewhere. Otherwise we should display:
```suggestion
        document.getElementById('webotsProgressMessage').innerHTML = 'An unknown error occurred during the loading.';
```

Note: occurred has two "r".